maxfw/utils/image_utils.py

```diff
@@ -0,0 +1,309 @@
+# standard lib
+from abc import ABC
+import io
+
+# other dependencies
+from flask import abort
+from PIL import Image
+import numpy as np
+
+
+class ImageProcessor(ABC):
```

Comment:
I don't think we need an `ABC` here since all the methods are implemented. Inheriting from `object` should be fine.

Reply:
Updated in 2db46a7.